src/shortcuts/__tests__/undo.ts

```diff
@@ -354,3 +354,35 @@ it('non-undoable actions are ignored', () => {
 
   expect(store.getState().inversePatches.length).toEqual(0)
 })
+
+it('undo redo importText action', () => {
```

Comment:
I added this test to the already existing https://github.com/cybersemics/em/blob/dev/src/shortcuts/__tests__/undo.ts. 

We follow the convention of naming the test file the same as the shortcut, reducer, or component that it's testing.